source/includes/fact-system-vars.rst

```diff
@@ -26,25 +28,144 @@
        - ``2``: For internal use only.
      - ``2``
      - ``0``
+
    * - ``max_nested_table_depth``
      - integer
      - Specifies the maximum number of unique nested field paths that
        ``mongosqld`` maps to a relational table for a collection.
      - ``50``
      - ``50``
+
    * - ``max_num_columns_per_table``
      - integer
      - The maximum number of unique fields that ``mongosqld`` maps to
        relational columns for a collection.
      - ``1000``
      - ``1000``
+
+   * - ``metrics_backend``
```

Comment:
pinging @sethpayne for an opinion on whether we want to document this one at all.